tests/test_logging/test_logging.py

```diff
@@ -4,65 +4,26 @@
 import pytest
```

Comment:
(I'll put this here because I don't know where else to put it. GH doesn't allow comments on deleted/empty files.)

Question: Why is the `test_logging/__init__.py` necessary?

Reply:
It's for pytest. Basically, if I don't put this there, I can't use `utils` 

see https://docs.pytest.org/en/stable/goodpractices.html#test-package-name

Reply:
got it, thanks